README_NEW.md

```diff
@@ -222,10 +235,14 @@ The above example contains all three extended key usages.
 	If needed, you can repeat this procedure to create multiple CSRs for different DSCs (on different maschines). When prompted, enter the necessary information (e.g. C= your jurisdicion (MUST), O = your Organisation (OPTIONAL), CN = non-empty and unique CN (MUST), …).
 
 	*Issue the certificate*
+  
 	Copy the CSR (DSC01csr.pem) to the folder where the private key of your CA is located. Open a command prompt and use the following command to issue the DSC (DSCcert.pem):
+
 	```
 	openssl x509 -req -in DSC01csr.pem -CA CAcert.pem -CAkey CAprivkey.key -CAcreateserial -days 730 -extensions ext -extfile DSC.conf -out DSCcert.pem
 	```
+  **Important Recommendations for Prod**: Please replace your CAcert.pem and CAprivkey.key which issued by your public Trusted Certificate Authority (CA) insterad of self-signged one.
```

Comment:
Please replace CAcert.pem and CAprivkey.key with signing material from an officially Trusted Certification Authority instead using your own self-signed one.

Reply:
Thanks Andreas for suggestion. I have done correction in my branch "readme_new_enhancement " 
https://github.com/WorldHealthOrganization/smart-trust/blob/da8d852d6c48e29a628afe4cda524673ab71606a/README_NEW.md?plain=1#L244

I kindly request to review and approve the same .